content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -24,15 +24,32 @@ This provides observability for libraries not covered by Datadog SDKs without ch
 
 Before adding OpenTelemetry instrumentation libraries, set the `DD_TRACE_OTEL_ENABLED` environment variable to `true`.
 
+Datadog SDKs provide an implementation of the OpenTelemetry API that submits spans to a Datadog Agent. To use the OpenTelemetry
+Instrumentations with Datadog SDKs ensure a Datadog Agent is [running][16].
+
 <div class="alert alert-warning">
 When replacing a Datadog instrumentation with its OpenTelemetry equivalent, disable the
 Datadog instrumentation to avoid duplicate spans in the trace.
 </div>
 
+<div class="alert alert-warning">
+Datadog SDKs implement the OpenTelemetry API by overriding the default implementations in the OpenTelemetry SDK. Using operations only supported by 
+OpenTelemetry SDK are not supported (ex: SpanProcessors, OTLP Trace Exporters).
```

Comment:
I suggest you change "Using operations only supported by 
OpenTelemetry SDK are not supported (ex: SpanProcessors, OTLP Trace Exporters)."
to something like,
"Items specific to the Opentelemetry SDK (ex: SpanProcessors, OTLP Trace Exporters) are not supported."